packages/perseus-linter/src/__tests__/rules.test.ts

```diff
@@ -532,4 +533,79 @@ describe("Individual lint rules tests", () => {
         "This is definitely okay. Yeah.",
         "$a == 3.  125$",
     ]);
+
+    test("Rule static-widget-in-question-stem allows static widgets in hints", () => {
+        const problems = testRule(staticWidgetInQuestionStem, "[[☃ radio 1]]", {
+            contentType: "exercise",
+            stack: ["hint"],
+            widgets: {
+                "radio 1": {
+                    options: {
+                        static: true,
+                    },
+                },
+            },
+        })
+
+        expect(problems).toBe(null)
+    });
```

Comment:
I originally wrote these tests using the `expectPass` etc. helper functions defined in this file. Then I realized that Jest wasn't running all the tests (or maybe Goland just wasn't displaying all the results?) because some of them had duplicate titles. In an ideal world, I think this file would be table tests.